fs/dafsserver/dafsserver.cpp

```diff
@@ -162,6 +162,69 @@ static ISecureSocket *createSecureSocket(ISocket *sock, bool disableClientCertVe
 }
 #endif
 
+//------------------------------------------------------------------------------
```

Comment:
ActiveSpanScope is very similar to ThreadedSpanScope described by Gavin here: https://hpccsystems.atlassian.net/jira/software/c/projects/HPCC/issues/HPCC-32982. I liked the name ActiveSpanScope because I believe the class has utility outside of multithreaded contexts, IE: time slicing. Is it worthwhile to move this out of dafilesrv into jtrace?

Reply:
yes.  These utility changes should be in jlib, ideally as separate PRs/requests.  It would be worth merging your other PR, and having a PR that implements an agreed solution to HPCC-32982, and then rebasing this PR on it.
I'm open to discussing what the different classes should be called.